packages/gatsby-source-shopify/src/create-operations.ts

```diff
@@ -68,6 +72,19 @@ export function createOperations(
         while (!finishedStatuses.includes(currentBulkOperation.status)) {
           await new Promise(resolve => setTimeout(resolve, 1000))
           currentBulkOperation = (await currentOperation()).currentBulkOperation
+
+          // add warning for CI environments
```

Comment:
@j0sh77 aren't there activities in Gatsby Cloud as well? What's different about CI vs local?

Reply:
To be honest, I'm not sure what qualifies to be shown in Gatsby Cloud or not. I'm not seeing anything around these long-running queries in Cloud currently. The site I'm testing with definitely has a long running query that I can reproduce locally.

@LekoArts would you happen to know if there's other qualifies an activity to show up?

Reply:
Maybe the activityTimer call needs an id or parentSpan to show up in the UI? 🤔 For example this activity timer shows up there https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/commands/build.ts#L358-L363

Reply:
Yeah, that could be. I think this warning is still a valuable add outside of the activity timer.

Reply:
![image](https://user-images.githubusercontent.com/5430474/181500751-b7eb204d-d9e0-4483-82c9-845c898617d0.png)

Reply:
I also added a slowdown. After the first few alerts, it starts increasing the time in which it warns.

Reply:
Ah I mean in the structured logs view. But since it's slowing down over time now it should be good :) I was just worried about getting a super long page in the structured view